apps/fluent-tester/src/FluentTester/TestComponents/Avatar/AvatarTest.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from 'react';
+import { Avatar } from '@fluentui/react-native';
+import { Stack } from '@fluentui-react-native/stack';
+import { stackStyle } from '../Common/styles';
+import { AVATAR_TESTPAGE } from './consts';
+import { Test, TestSection, PlatformStatus } from '../Test';
+
+const avatar: React.FunctionComponent<{}> = () => {
+  return (
+    <Stack style={stackStyle}>
+      <Avatar primaryText="John Smith" />
+      <Avatar primaryText="John Smith" avatarStyle="square" />
+      <Avatar primaryText="John Smith" presence="available" />
+      <Avatar primaryText="John Smith" secondaryText="johnsmith@microsoft.com" presence="outOfOffice" />
+      <Avatar secondaryText="johnsmith@microsoft.com" presence="outOfOffice" />
+    </Stack>
+  );
+};
+
+const stylizedAvatar: React.FunctionComponent<{}> = () => {
+  const CustomizedAvatar = Avatar.customize({
+    size: 'large',
+  });
+
+  return (
+    <Stack style={stackStyle}>
+      <CustomizedAvatar primaryText="John Smith" />
+      <CustomizedAvatar primaryText="John Smith" avatarStyle="square" />
+      <CustomizedAvatar primaryText="John Smith" presence="available" />
+      <CustomizedAvatar primaryText="John Smith" secondaryText="johnsmith@microsoft.com" presence="outOfOffice" />
+      <CustomizedAvatar secondaryText="johnsmith@microsoft.com" presence="outOfOffice" />
```

Comment:
we should add phone number case or non English case? just a thought

Reply:
We support phone numbers for secondaryText? I ca take a look at the cases we have in the native test app

Reply:
phone number as primary text for phone sign in

Reply:
OK, we don't have those examples in the iOS test app :D

Reply:
I think i'll leave the "better test cases to a separate PR, because I think I can steal a bunch of stuff from the PersonaCoin test cases